src/ctapipe/calib/camera/tests/test_extractor.py

```diff
@@ -0,0 +1,141 @@
+"""
+Tests for StatisticsExtractor and related functions
+"""
+
+import numpy as np
+import pytest
+from astropy.table import Table
+from astropy.time import Time
+
+from ctapipe.calib.camera.extractor import PlainExtractor, SigmaClippingExtractor
```

Comment:
in real use (outside of tests) do we really want to import this from such a deep module? Should we maybe expose it at the `ctapipe.calib` level in `__init__.py`?